tests/test_csc.py

```diff
@@ -327,27 +330,32 @@ async def publish_mountEnconders(ntimes=5):
                     '%s' % harness.aos_remote.evt_detailedState.callback.call_args_list
                 )
 
-                # disable CSC
+                logger.debug("Disable ATAOS CSC")
                 await harness.aos_remote.cmd_disable.start(timeout=timeout)
 
         # Run test getting the telescope position
         asyncio.get_event_loop().run_until_complete(doit(get_tel_pos=True))
+        logger.debug("test getting the telescope position - COMPLETE")
 
         # Run test getting the telescope position while exposing
         asyncio.get_event_loop().run_until_complete(doit(get_tel_pos=True,
                                                          while_exposing=True))
+        logger.debug("Run test getting the telescope position while exposing - COMPLETE")
 
         # Run for specified location
         asyncio.get_event_loop().run_until_complete(doit(get_tel_pos=False))
+        logger.debug("Run for specified location - COMPLETE")
 
     def test_offsets(self):
-        """Test applyFocusOffset command."""
+        """Test offset command (which applies offsets to the models)"""
 
         async def doit():
 
             async with Harness() as harness:
 
-                await salobj.set_summary_state(harness.aos_remote, salobj.State.ENABLED)
+                # if there is nothing (atpneumatics/atspectrograph) sending events then this command times out
```

Comment:
This line must be violating pep8 maximum line length no? It is longer than 80 characters.

Reply:
No, 110 is the limit... and this is exactly 110...